_posts/2023-01-24-CharlieBroderick-introduction-meme.md

```diff
@@ -0,0 +1,7 @@
+---
+title:  minicomputervsPCs
+author: charlie.md
```

Comment:
This needs to match the short_name in your author profile. So, change this to author: charlie.